artistools/lightcurve/viewingangleanalysis.py

```diff
@@ -390,7 +406,7 @@ def make_plot_test_viewing_angle_fit(
 
     plt.xlabel("Time Since Explosion [d]")
     plt.gca().invert_yaxis()
-    plt.xlim(0, 40)
+    plt.xlim(args.timemin - 1, args.timemax + 1)
```

Comment:
How was the value of 1 chosen here? if you want some extra room around the room, scale it proportionally, e.g.
`plt.xlim(args.timemin / 1.1, args.timemax  * 1.1)`